.travis.yml

```diff
@@ -141,7 +141,7 @@ matrix:
     - rvm: 2.4.4
       env: RAILS_VERSION='~> 3.1.12'
     - rvm: 2.4.4
-      env: RAILS_VERSION=5.2.0.rc1
+      env: RAILS_VERSION=5.2.0
```

Comment:
Ah hem, this is in the wrong place, it actually prevents `5.2.0` from running on `2.4.4`, it should be include with an entry for each version it supports

Reply:
Indeed this appears to be at end of some exclusions. But, why is it here? I don’t see anything that would test against 5.2 in the first place. 

Looks like I should have added entries under the include on line 60.

Reply:
Yes it's in the wrong place and no one noticed :)